skills/types/all_test.go

```diff
@@ -0,0 +1,36 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package types_test
+
+import (
+	"github.com/ubuntu-core/snappy/skills/types"
+	. "github.com/ubuntu-core/snappy/testutil"
+
+	. "gopkg.in/check.v1"
+)
+
+type AllSuite struct{}
+
+var _ = Suite(&AllSuite{})
+
+func (s *AllSuite) TestAllTypes(c *C) {
+	allTypes := types.AllTypes()
+	c.Check(allTypes, Contains, &types.BoolFileType{})
```

Comment:
Sorry for being "mr-smarty-pants" here: `c.Check(allTypes, DeepEquals, []skills.Type{&types.BoolFileType{}})` would  have worked as well. But now Contains knows about interfaces which is nice as well!